### PR TITLE
final PR

### PR DESCRIPTION
**Pull request by Saddamshah**
Please add my files I've completed all the projects as well as blogs

### CODE REVIEW

File: content/participants/ttjc@Saddamshah/index.md [hunk 1]
```diff
@@ -102,3 +102,58 @@ Email Id: admin@admin.com
 Password: 12345
```
Comment (gkanishk) [inline content/participants/ttjc@Saddamshah/index.md:102]: shorten this part only provide a small description and you can provide description inside readme file.

**Comment by Saddamshah** [comment]
Hii Mentors, I've made changes and edited the finalist's file as per the requirement. 
Kindly check the final file.

File: content/finalist/ttjc@Saddamshah/index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+##  [ Project No 4 Data Visualization Map ]
+_description_ A Data Visualization that show covid cases on map.
+
+_hosted link_ https://data-visualization-covid.herokuapp.com/
+_github link_ https://github.com/Saddamshah/Data-Visualization-covid-
+
+##  [ Project No 5 Currency Converter ]
+_description_ A Currency Converter app using react.
+
+ _hosted link_ https://app.netlify.com/sites/currency-converter-project
```
Comment (gkanishk) [inline content/finalist/ttjc@Saddamshah/index.md:75]: You have not provided the correct hosted link for the project kindly check them.

File: content/finalist/ttjc@Saddamshah/index.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+---
+title: Saddam Shah
+type: "finalist"
+name: "Saddam Shah"
+portfolio: "https://saddamshah.netlify.app/"
+github-link: "https://github.com/Saddamshah/"
+linkedin-link: "https://www.linkedin.com/in/saddam-shah/"
+looking-for: "job"
+image-link: "https://avatars0.githubusercontent.com/u/46083623?s=460&u=d2e51ad2754d31669119720791a3f2a38a05b112&v=4"
```
Comment (gkanishk) [inline content/finalist/ttjc@Saddamshah/index.md:9]: As per previous comments I advised to use correct format i.e "image_link" so please change all the variables as the same format

**Comment by Saddamshah** [comment]
I made the correction kindly check.
Thank You.

**Pull request merged**